drivers/wifi/nrf700x/src/fmac_main.c

```diff
@@ -483,11 +483,7 @@ enum nrf_wifi_status nrf_wifi_fmac_dev_add_zep(struct nrf_wifi_drv_priv_zep *drv
 	enum nrf_wifi_status status = NRF_WIFI_STATUS_FAIL;
 	struct nrf_wifi_ctx_zep *rpu_ctx_zep = NULL;
 	void *rpu_ctx = NULL;
-#if defined(CONFIG_BOARD_NRF7001)
-	enum op_band op_band = BAND_24G;
-#else /* CONFIG_BOARD_NRF7001 */
-	enum op_band op_band = BAND_ALL;
-#endif /* CONFIG_BOARD_NRF7001 */
+	enum op_band op_band = CONFIG_NRF_WIFI_OP_BAND;
```

Comment:
Is the Kconfig and enum matching?

Reply:
Yes, Current enum order is matching with Kconfig. There are 2 options available with enum, BAND_ALL and BAND_24G. Other Kconfig are for further enhancement.